Packs/SMB/Integrations/SMB_v2/SMB_v2.py

```diff
@@ -186,11 +188,14 @@ def main():
         try:
             client_guid = uuid.UUID(client_guid)
         except ValueError:
-            demisto.info(f'Failed to convert {client_guid} to a valid UUID string. Using a random generated UUID instead')
+            demisto.info(
+                f'Failed to convert {client_guid} to a valid UUID string. Using a random generated UUID instead')
             client_guid = None
 
-    smbclient.ClientConfig(username=user, password=password, require_secure_negotiate=verify, domain_controller=dc,
-                           client_guid=client_guid)
+    # Temporary workaround to an issue in the smbprotocol package.
```

Comment:
add a link to the github issue

Reply:
It was closed due to the workaround. Should be resolved in the next release. I'll keep track of it.